src/Whoops/Util/TemplateHelper.php

```diff
@@ -59,6 +63,42 @@ public function escapeButPreserveUris($raw)
     }
 
     /**
+     * Format the given value into a human readable string.
+     *
+     * @param  mixed $value
+     * @return string
+     */
+    public function dump($value)
+    {
+        if (class_exists('Symfony\Component\VarDumper\Cloner\VarCloner')) {
+            $cloner = new VarCloner();
+            if ('cli' === PHP_SAPI) {
```

Comment:
I think it's plenty reliable. The entire internet of PHP devs use this, it won't be broken.
At least for our purposes it's not vital enough to worry about the risk.